test/command.js

```diff
@@ -1,15 +1,17 @@
-import path from 'path';
+import path from 'node:path';
+import process from 'node:process';
+import {fileURLToPath} from 'node:url';
 import test from 'ava';
-import execa from '..';
+import * as execa from '../index.js';
```

Comment:
Could we use named imports instead?

Reply:
See PR description and Sindre's comment but the reason I did a namespace import here was to keep the diff smaller. Will probably have to change anyway.

Reply:
Yes, I believe this would be better to use named imports there, also for consistency with the other test files, despite the fact that it will make the PR diff bigger.